docs: improved Dynatrace integration guide for metrics (#2596)
kyma-project/telemetry-manager: docs/user/integration/dynatrace/README.md

            output:
                otlp:
                    endpoint:
                        value: http://dynakube-telemetry-ingest.${DYNATRACE_NS}:4317
        EOF
        ```

> - If your application pushes metrics in delta temporality with OTLP (and you don't want to collect metrics using the MetricPipeline `istio` and `prometheus` input), push the metrics directly to Dynatrace. This setup will ship data directly to the Dynatrace backend and will avoid unneccessary processing in additional components.

Reviewer: ```suggestion
- If your application pushes OTLP metrics in delta temporality, and you don't use the MetricPipeline's `istio` or `prometheus` inputs, push the metrics directly to Dynatrace. Shipping data directly to the Dynatrace backend prevents unnecessary processing by additional components.
```

>   To use this setup, you must explicitly enable the "delta" aggregation temporality as prefered tempoarility in your applications. You cannot enable additional inputs for the MetricPipeline because these produce metrics with "cumulative" temporability.

Reviewer: ```suggestion
  To use this setup, you must explicitly enable the "delta" aggregation temporality as preferred temporality in your applications. You cannot enable additional inputs for the MetricPipeline because these produce metrics with "cumulative" temporality.
```
